annotations/views/user_views.py

```diff
@@ -227,3 +229,19 @@ def get_weekly_annotations(self, user):
 				'count': weekly_annotations.get(week.datetime, 0)
 			})
 		return result
+	
+@api_view(["POST"])
+def create(request):
```

Comment:
Is this necessary? Whats wrong with /users?

Reply:
Seperating user creation and signup. In user creation we need not return the access and refresh tokens, for signup UI is setting those in local storage. So the signup API creates user and then returns the refresh token for users.